package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@code4ro/reusable-components",
+  "version": "1.0.0",
+  "description": "Component library for code4ro",
+  "private": false,
+  "sideEffects": false,
+  "devDependencies": {
+    "@babel/core": "^7.11.4",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@babel/plugin-proposal-export-default-from": "^7.10.4",
+    "@babel/preset-env": "^7.11.0",
+    "@babel/preset-react": "^7.10.4",
+    "@rollup/plugin-image": "^2.0.5",
+    "@storybook/addon-actions": "^6.0.16",
+    "@storybook/addon-console": "^1.2.1",
+    "@storybook/addon-docs": "^6.0.16",
+    "@storybook/addon-essentials": "^6.0.16",
```

Comment:
`Docs, Controls, Actions, Viewport, Backgrounds,Toolbars & globals` are already included in `@storybook/addon-essentials`, so we can remove `@storybook/addon-actions`, `@storybook/addon-docs` and `@storybook/addon-viewport`